index.bs

```diff
@@ -549,15 +549,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Authenticator</dfn>
 : <dfn>[WAA]</dfn>
-:: A cryptographic entity used by a [=[WAC]=] to (i) generate a [=public key credential=] and register it with a [=[RP]=],
-    and (ii) [=authentication|authenticate=] by potentially [=user verification|verifying the user=], and then 
-    cryptographically signing and returning, in the form of an [=Authentication Assertion=], 
-    a challenge and other data presented by a [=[WRP]=] (in concert with the [=[WAC]=]).
+:: A cryptographic entity, existing in hardware or software, that is capable of being used by a [=[WAC]=] to 
+    [=registration|register=] a user with a given [=[RP]=] and [=Authentication Assertion|asserting ownership=] of the 
+    registered [=public key credential=] when requested by the [=[RP]=]. [=Authenticators=] can report information regarding
+    their [=authenticator types|type=] and security characteristics via [=attestation=] during [=registration=].
```

Comment:
Although better than previously, the first sentence still feels a bit dense. Suggestions:

- s/is capable of being/can be/
- s/and [=Authentication Assertion|asserting ownership=]/and later [=Authentication Assertion|assert ownership=]/
- Maybe we can also drop "by a [=[WAC]=]"?

Result:

>A cryptographic entity, existing in hardware or software, that can be used to [=registration|register=] a user with a given [=[RP]=] and later [=Authentication Assertion|assert ownership=] of the registered [=public key credential=] when requested by the [=[RP]=].